### ISSUE TITLE
"make local" failing on Centos7

### ISSUE THREAD

**Issue by mxMiles** [Islandora-Devops/isle-dc#168]
Steps taken:

- git clone https://github.com/islandora-devops/isle-dc
- cd isle-dc
- make local 

```
arclight@grysbok:~> pwd
/home/arclight
arclight@grysbok:~> git clone https://github.com/islandora-devops/isle-dc
Cloning into 'isle-dc'...
remote: Enumerating objects: 2268, done.
remote: Counting objects: 100% (147/147), done.
remote: Compressing objects: 100% (68/68), done.
remote: Total 2268 (delta 82), reused 135 (delta 76), pack-reused 2121
Receiving objects: 100% (2268/2268), 45.81 MiB | 64.96 MiB/s, done.
Resolving deltas: 100% (1245/1245), done.
arclight@grysbok:~> cd isle-dc/
arclight@grysbok:~/isle-dc> make local
make[1]: Entering directory '/home/arclight/isle-dc'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  5596  100  5596    0     0  31629      0 --:--:-- --:--:-- --:--:-- 31795
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1704  100  1704    0     0   9674      0 --:--:-- --:--:-- --:--:--  9737
make[1]: Leaving directory '/home/arclight/isle-dc'
make[1]: Entering directory '/home/arclight/isle-dc'
make[1]: Leaving directory '/home/arclight/isle-dc'
make[1]: Entering directory '/home/arclight/isle-dc'
docker-compose pull
Pulling activemq   ... done
Pulling alpaca     ... done
Pulling blazegraph ... done
Pulling cantaloupe ... done
Pulling fcrepo     ... done
Pulling fits       ... done
Pulling crayfits   ... done
Pulling homarus    ... done
Pulling houdini    ... done
Pulling hypercube  ... done
Pulling mariadb    ... done
Pulling matomo     ... done
Pulling milliner   ... done
Pulling recast     ... done
Pulling solr       ... done
Pulling traefik    ... done
Pulling drupal     ... done
Pulling watchtower ... done
make[1]: Leaving directory '/home/arclight/isle-dc'
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-container-environment-00-init.sh: executing...
[cont-init.d] 00-container-environment-00-init.sh: exited 0.
[cont-init.d] 00-container-environment-01-override-nginx.sh: executing...
[cont-init.d] 00-container-environment-01-override-nginx.sh: exited 0.
[cont-init.d] 00-container-environment-01-override-php.sh: executing...
[cont-init.d] 00-container-environment-01-override-php.sh: exited 0.
[cont-init.d] 00-container-environment-02-database-defaults.sh: executing...
[cont-init.d] 00-container-environment-02-database-defaults.sh: exited 0.
[cont-init.d] 01-confd-render-templates.sh: executing...
[cont-init.d] 01-confd-render-templates.sh: exited 0.
[cont-init.d] 02-fpm-install.sh: executing...
[cont-init.d] 02-fpm-install.sh: exited 0.
[cont-init.d] 02-ngnix-install.sh: executing...
[cont-init.d] 02-ngnix-install.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[14-Jul-2021 14:07:35] NOTICE: fpm is running, pid 722
[14-Jul-2021 14:07:35] NOTICE: ready to handle connections
Creating a "drupal/recommended-project:^9.1" project at "/tmp/codebase"
Installing drupal/recommended-project (9.2.1)
  - Downloading drupal/recommended-project (9.2.1)
  - Installing drupal/recommended-project (9.2.1): Extracting archive
Created project in /tmp/codebase
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Package operations: 63 installs, 0 updates, 0 removals
  - Downloading composer/installers (v1.11.0)
  - Downloading drupal/core-composer-scaffold (9.2.1)
  - Downloading drupal/core-project-message (9.2.1)
  - Downloading typo3/phar-stream-wrapper (v3.1.6)
  - Downloading symfony/polyfill-mbstring (v1.23.0)
  - Downloading symfony/polyfill-ctype (v1.23.0)
  - Downloading twig/twig (v2.14.6)
  - Downloading symfony/yaml (v4.4.25)
  - Downloading symfony/polyfill-php80 (v1.23.0)
  - Downloading symfony/var-dumper (v5.3.0)
  - Downloading symfony/translation-contracts (v2.4.0)
  - Downloading symfony/validator (v4.4.25)
  - Downloading symfony/translation (v4.4.25)
  - Downloading psr/container (1.1.1)
  - Downloading symfony/service-contracts (v2.4.0)
  - Downloading symfony/serializer (v4.4.25)
  - Downloading symfony/routing (v4.4.25)
  - Downloading symfony/polyfill-php72 (v1.23.0)
  - Downloading symfony/polyfill-intl-normalizer (v1.23.0)
  - Downloading symfony/polyfill-intl-idn (v1.23.0)
  - Downloading symfony/deprecation-contracts (v2.4.0)
  - Downloading symfony/mime (v5.3.0)
  - Downloading symfony/http-foundation (v4.4.25)
  - Downloading psr/http-message (1.0.1)
  - Downloading symfony/psr-http-message-bridge (v2.1.0)
  - Downloading symfony/process (v4.4.25)
  - Downloading symfony/polyfill-iconv (v1.23.0)
  - Downloading symfony/polyfill-php73 (v1.23.0)
  - Downloading symfony/http-client-contracts (v2.4.0)
  - Downloading symfony/event-dispatcher-contracts (v1.1.9)
  - Downloading symfony/event-dispatcher (v4.4.25)
  - Downloading psr/log (1.1.4)
  - Downloading symfony/debug (v4.4.25)
  - Downloading symfony/error-handler (v4.4.25)
  - Downloading symfony/http-kernel (v4.4.25)
  - Downloading symfony/dependency-injection (v4.4.25)
  - Downloading symfony/console (v4.4.25)
  - Downloading symfony-cmf/routing (2.3.3)
  - Downloading stack/builder (v1.0.6)
  - Downloading ralouphie/getallheaders (3.0.3)
  - Downloading psr/http-factory (1.0.1)
  - Downloading psr/cache (1.0.1)
  - Downloading pear/pear_exception (v1.0.2)
  - Downloading pear/console_getopt (v1.4.3)
  - Downloading pear/pear-core-minimal (v1.10.10)
  - Downloading pear/archive_tar (1.4.13)
  - Downloading masterminds/html5 (2.7.4)
  - Downloading laminas/laminas-zendframework-bridge (1.2.0)
  - Downloading laminas/laminas-stdlib (3.3.1)
  - Downloading laminas/laminas-escaper (2.7.0)
  - Downloading laminas/laminas-feed (2.14.1)
  - Downloading laminas/laminas-diactoros (2.6.0)
  - Downloading guzzlehttp/psr7 (1.8.2)
  - Downloading guzzlehttp/promises (1.4.1)
  - Downloading guzzlehttp/guzzle (6.5.5)
  - Downloading doctrine/lexer (1.2.1)
  - Downloading egulias/email-validator (2.1.25)
  - Downloading doctrine/annotations (1.13.1)
  - Downloading doctrine/reflection (1.2.2)
  - Downloading composer/semver (3.2.5)
  - Downloading asm89/stack-cors (1.3.0)
  - Downloading drupal/core (9.2.1)
  0/62 [>---------------------------]   0%
  8/62 [===>------------------------]  12%
 26/62 [===========>----------------]  41%
 39/62 [=================>----------]  62%
 52/62 [=======================>----]  83%
 61/62 [===========================>]  98%
 62/62 [============================] 100%  - Installing composer/installers (v1.11.0): Extracting archive
  - Installing drupal/core-composer-scaffold (9.2.1): Extracting archive
  - Installing drupal/core-project-message (9.2.1): Extracting archive
  - Installing typo3/phar-stream-wrapper (v3.1.6): Extracting archive
  - Installing symfony/polyfill-mbstring (v1.23.0): Extracting archive
  - Installing symfony/polyfill-ctype (v1.23.0): Extracting archive
  - Installing twig/twig (v2.14.6): Extracting archive
  - Installing symfony/yaml (v4.4.25): Extracting archive
  - Installing symfony/polyfill-php80 (v1.23.0): Extracting archive
  - Installing symfony/var-dumper (v5.3.0): Extracting archive
  - Installing symfony/translation-contracts (v2.4.0): Extracting archive
  - Installing symfony/validator (v4.4.25): Extracting archive
  - Installing symfony/translation (v4.4.25): Extracting archive
  - Installing psr/container (1.1.1): Extracting archive
  - Installing symfony/service-contracts (v2.4.0): Extracting archive
  - Installing symfony/serializer (v4.4.25): Extracting archive
  - Installing symfony/routing (v4.4.25): Extracting archive
  - Installing symfony/polyfill-php72 (v1.23.0): Extracting archive
  - Installing symfony/polyfill-intl-normalizer (v1.23.0): Extracting archive
  - Installing symfony/polyfill-intl-idn (v1.23.0): Extracting archive
  - Installing symfony/deprecation-contracts (v2.4.0): Extracting archive
  - Installing symfony/mime (v5.3.0): Extracting archive
  - Installing symfony/http-foundation (v4.4.25): Extracting archive
  - Installing psr/http-message (1.0.1): Extracting archive
  - Installing symfony/psr-http-message-bridge (v2.1.0): Extracting archive
  - Installing symfony/process (v4.4.25): Extracting archive
  - Installing symfony/polyfill-iconv (v1.23.0): Extracting archive
  - Installing symfony/polyfill-php73 (v1.23.0): Extracting archive
  - Installing symfony/http-client-contracts (v2.4.0): Extracting archive
  - Installing symfony/event-dispatcher-contracts (v1.1.9): Extracting archive
  - Installing symfony/event-dispatcher (v4.4.25): Extracting archive
  - Installing psr/log (1.1.4): Extracting archive
  - Installing symfony/debug (v4.4.25): Extracting archive
  - Installing symfony/error-handler (v4.4.25): Extracting archive
  - Installing symfony/http-kernel (v4.4.25): Extracting archive
  - Installing symfony/dependency-injection (v4.4.25): Extracting archive
  - Installing symfony/console (v4.4.25): Extracting archive
  - Installing symfony-cmf/routing (2.3.3): Extracting archive
  - Installing stack/builder (v1.0.6): Extracting archive
  - Installing ralouphie/getallheaders (3.0.3): Extracting archive
  - Installing psr/http-factory (1.0.1): Extracting archive
  - Installing psr/cache (1.0.1): Extracting archive
  - Installing pear/pear_exception (v1.0.2): Extracting archive
  - Installing pear/console_getopt (v1.4.3): Extracting archive
  - Installing pear/pear-core-minimal (v1.10.10): Extracting archive
  - Installing pear/archive_tar (1.4.13): Extracting archive
  - Installing masterminds/html5 (2.7.4): Extracting archive
  - Installing laminas/laminas-zendframework-bridge (1.2.0): Extracting archive
  - Installing laminas/laminas-stdlib (3.3.1): Extracting archive
  - Installing laminas/laminas-escaper (2.7.0): Extracting archive
  - Installing laminas/laminas-feed (2.14.1): Extracting archive
  - Installing laminas/laminas-diactoros (2.6.0): Extracting archive
  - Installing guzzlehttp/psr7 (1.8.2): Extracting archive
  - Installing guzzlehttp/promises (1.4.1): Extracting archive
  - Installing guzzlehttp/guzzle (6.5.5): Extracting archive
  - Installing doctrine/lexer (1.2.1): Extracting archive
  - Installing egulias/email-validator (2.1.25): Extracting archive
  - Installing doctrine/annotations (1.13.1): Extracting archive
  - Installing doctrine/reflection (1.2.2): Extracting archive
  - Installing composer/semver (3.2.5): Extracting archive
  - Installing asm89/stack-cors (1.3.0): Extracting archive
  - Installing drupal/core (9.2.1): Extracting archive
  - Installing drupal/core-recommended (9.2.1)
  0/50 [>---------------------------]   0%
  8/50 [====>-----------------------]  16%
 25/50 [==============>-------------]  50%
 40/50 [======================>-----]  80%
 49/50 [===========================>]  98%
 50/50 [============================] 100%Package doctrine/reflection is abandoned, you should avoid using it. Use roave/better-reflection instead.
Generating autoload files
40 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
Scaffolding files for drupal/core:
  - Copy [project-root]/.editorconfig from assets/scaffold/files/editorconfig
  - Copy [project-root]/.gitattributes from assets/scaffold/files/gitattributes
  - Copy [web-root]/.csslintrc from assets/scaffold/files/csslintrc
  - Copy [web-root]/.eslintignore from assets/scaffold/files/eslintignore
  - Copy [web-root]/.eslintrc.json from assets/scaffold/files/eslintrc.json
  - Copy [web-root]/.ht.router.php from assets/scaffold/files/ht.router.php
  - Copy [web-root]/.htaccess from assets/scaffold/files/htaccess
  - Copy [web-root]/example.gitignore from assets/scaffold/files/example.gitignore
  - Copy [web-root]/index.php from assets/scaffold/files/index.php
  - Copy [web-root]/INSTALL.txt from assets/scaffold/files/drupal.INSTALL.txt
  - Copy [web-root]/README.md from assets/scaffold/files/drupal.README.md
  - Copy [web-root]/robots.txt from assets/scaffold/files/robots.txt
  - Copy [web-root]/update.php from assets/scaffold/files/update.php
  - Copy [web-root]/web.config from assets/scaffold/files/web.config
  - Copy [web-root]/sites/README.txt from assets/scaffold/files/sites.README.txt
  - Copy [web-root]/sites/development.services.yml from assets/scaffold/files/development.services.yml
  - Copy [web-root]/sites/example.settings.local.php from assets/scaffold/files/example.settings.local.php
  - Copy [web-root]/sites/example.sites.php from assets/scaffold/files/example.sites.php
  - Copy [web-root]/sites/default/default.services.yml from assets/scaffold/files/default.services.yml
  - Copy [web-root]/sites/default/default.settings.php from assets/scaffold/files/default.settings.php
  - Copy [web-root]/modules/README.txt from assets/scaffold/files/modules.README.txt
  - Copy [web-root]/profiles/README.txt from assets/scaffold/files/profiles.README.txt
  - Copy [web-root]/themes/README.txt from assets/scaffold/files/themes.README.txt
  * Homepage: https://www.drupal.org/project/drupal
  * Support:
    * docs: https://www.drupal.org/docs/user_guide/en/index.html
    * chat: https://www.drupal.org/node/314178

  Congratulations, you’ve installed the Drupal codebase
  from the drupal/recommended-project template!


Next steps:
  * Install the site: https://www.drupal.org/docs/8/install
  * Read the user guide: https://www.drupal.org/docs/user_guide/en/index.html
  * Get support: https://www.drupal.org/support
  * Get involved with the Drupal community:
      https://www.drupal.org/getting-involved
  * Remove the plugin that prints this message:
      composer remove drupal/core-project-message
  * Homepage: https://www.drupal.org/project/drupal
  * Support:
    * docs: https://www.drupal.org/docs/user_guide/en/index.html
    * chat: https://www.drupal.org/node/314178
./composer.json has been updated
Running composer update islandora/islandora
Loading composer repositories with package information
Updating dependencies
Lock file operations: 44 installs, 0 updates, 0 removals
  - Locking doctrine/cache (2.1.0)
  - Locking doctrine/dbal (2.13.2)
  - Locking doctrine/deprecations (v0.5.3)
  - Locking doctrine/event-manager (1.1.1)
  - Locking drupal/config_update (1.7.0)
  - Locking drupal/context (4.0.0-beta6)
  - Locking drupal/eva (2.1.0)
  - Locking drupal/features (3.12.0)
  - Locking drupal/file_replace (1.2.0)
  - Locking drupal/filehash (1.5.0)
  - Locking drupal/flysystem (2.0.0-alpha1)
  - Locking drupal/jwt (1.0.0-rc1)
  - Locking drupal/key (1.14.0)
  - Locking drupal/migrate_plus (5.1.0)
  - Locking drupal/migrate_source_csv (3.4.0)
  - Locking drupal/migrate_tools (5.0.0)
  - Locking drupal/prepopulate (2.3.0)
  - Locking drupal/search_api (1.19.0)
  - Locking drupal/token (1.9.0)
  - Locking easyrdf/easyrdf (1.1.1)
  - Locking firebase/php-jwt (v5.4.0)
  - Locking islandora/chullo (dev-dev d563d5e)
  - Locking islandora/crayfish-commons (dev-dev 41f9482)
  - Locking islandora/islandora (dev-8.x-1.x 85dae5c)
  - Locking islandora/jsonld (dev-8.x-1.x dfd99c4)
  - Locking league/csv (9.7.1)
  - Locking league/flysystem (1.1.4)
  - Locking league/flysystem-replicate-adapter (1.0.1)
  - Locking league/mime-type-detection (1.7.0)
  - Locking ml/iri (1.1.4)
  - Locking ml/json-ld (1.2.0)
  - Locking monolog/monolog (1.26.1)
  - Locking namshi/jose (7.2.3)
  - Locking pimple/pimple (v3.4.0)
  - Locking silex/silex (v2.3.0)
  - Locking stomp-php/stomp-php (4.6.1)
  - Locking symfony/polyfill-intl-grapheme (v1.23.0)
  - Locking symfony/polyfill-php56 (v1.20.0)
  - Locking symfony/property-access (v5.3.0)
  - Locking symfony/property-info (v5.3.1)
  - Locking symfony/security (v4.4.26)
  - Locking symfony/string (v5.3.3)
  - Locking twistor/flysystem-stream-wrapper (v1.0.9)
  - Locking twistor/stream-util (v1.0.1)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 44 installs, 0 updates, 0 removals
  - Downloading doctrine/event-manager (1.1.1)
  - Downloading doctrine/deprecations (v0.5.3)
  - Downloading doctrine/cache (2.1.0)
  - Downloading doctrine/dbal (2.13.2)
  - Downloading drupal/config_update (1.7.0)
  - Downloading drupal/key (1.14.0)
  - Downloading firebase/php-jwt (v5.4.0)
  - Downloading ml/iri (1.1.4)
  - Downloading ml/json-ld (1.2.0)
  - Downloading easyrdf/easyrdf (1.1.1)
  - Syncing islandora/chullo (dev-dev d563d5e) into cache
  - Downloading stomp-php/stomp-php (4.6.1)
  - Syncing islandora/jsonld (dev-8.x-1.x dfd99c4) into cache
  - Downloading symfony/polyfill-intl-grapheme (v1.23.0)
  - Downloading symfony/string (v5.3.3)
  - Downloading symfony/property-info (v5.3.1)
  - Downloading symfony/property-access (v5.3.0)
  - Downloading symfony/security (v4.4.26)
  - Downloading pimple/pimple (v3.4.0)
  - Downloading silex/silex (v2.3.0)
  - Downloading namshi/jose (7.2.3)
  - Downloading monolog/monolog (1.26.1)
  - Syncing islandora/crayfish-commons (dev-dev 41f9482) into cache
  - Downloading drupal/token (1.9.0)
  - Downloading drupal/search_api (1.19.0)
  - Downloading drupal/prepopulate (2.3.0)
  - Downloading drupal/migrate_plus (5.1.0)
  - Downloading drupal/migrate_tools (5.0.0)
  - Downloading league/csv (9.7.1)
  - Downloading drupal/migrate_source_csv (3.4.0)
  - Downloading drupal/jwt (1.0.0-rc1)
  - Downloading twistor/stream-util (v1.0.1)
  - Downloading league/mime-type-detection (1.7.0)
  - Downloading league/flysystem (1.1.4)
  - Downloading twistor/flysystem-stream-wrapper (v1.0.9)
  - Downloading league/flysystem-replicate-adapter (1.0.1)
  - Downloading drupal/flysystem (2.0.0-alpha1)
  - Downloading drupal/filehash (1.5.0)
  - Downloading drupal/file_replace (1.2.0)
  - Downloading drupal/features (3.12.0)
  - Downloading drupal/eva (2.1.0)
  - Downloading drupal/context (4.0.0-beta6)
  - Syncing islandora/islandora (dev-8.x-1.x 85dae5c) into cache
  0/39 [>---------------------------]   0%
  2/39 [=>--------------------------]   5%
 15/39 [==========>-----------------]  38%
 34/39 [========================>---]  87%
 39/39 [============================] 100%  - Installing doctrine/event-manager (1.1.1): Extracting archive
  - Installing doctrine/deprecations (v0.5.3): Extracting archive
  - Installing doctrine/cache (2.1.0): Extracting archive
  - Installing doctrine/dbal (2.13.2): Extracting archive
  - Installing drupal/config_update (1.7.0): Extracting archive
  - Installing drupal/key (1.14.0): Extracting archive
  - Installing firebase/php-jwt (v5.4.0): Extracting archive
  - Installing ml/iri (1.1.4): Extracting archive
  - Installing ml/json-ld (1.2.0): Extracting archive
  - Installing easyrdf/easyrdf (1.1.1): Extracting archive
  - Installing islandora/chullo (dev-dev d563d5e): Cloning d563d5e48e from cache
  - Installing stomp-php/stomp-php (4.6.1): Extracting archive
  - Installing islandora/jsonld (dev-8.x-1.x dfd99c4): Cloning dfd99c4c1b from cache
  - Installing symfony/polyfill-intl-grapheme (v1.23.0): Extracting archive
  - Installing symfony/string (v5.3.3): Extracting archive
  - Installing symfony/property-info (v5.3.1): Extracting archive
  - Installing symfony/property-access (v5.3.0): Extracting archive
  - Installing symfony/security (v4.4.26): Extracting archive
  - Installing pimple/pimple (v3.4.0): Extracting archive
  - Installing silex/silex (v2.3.0): Extracting archive
  - Installing symfony/polyfill-php56 (v1.20.0)
  - Installing namshi/jose (7.2.3): Extracting archive
  - Installing monolog/monolog (1.26.1): Extracting archive
  - Installing islandora/crayfish-commons (dev-dev 41f9482): Cloning 41f94824d5 from cache
  - Installing drupal/token (1.9.0): Extracting archive
  - Installing drupal/search_api (1.19.0): Extracting archive
  - Installing drupal/prepopulate (2.3.0): Extracting archive
  - Installing drupal/migrate_plus (5.1.0): Extracting archive
  - Installing drupal/migrate_tools (5.0.0): Extracting archive
  - Installing league/csv (9.7.1): Extracting archive
  - Installing drupal/migrate_source_csv (3.4.0): Extracting archive
  - Installing drupal/jwt (1.0.0-rc1): Extracting archive
  - Installing twistor/stream-util (v1.0.1): Extracting archive
  - Installing league/mime-type-detection (1.7.0): Extracting archive
  - Installing league/flysystem (1.1.4): Extracting archive
  - Installing twistor/flysystem-stream-wrapper (v1.0.9): Extracting archive
  - Installing league/flysystem-replicate-adapter (1.0.1): Extracting archive
  - Installing drupal/flysystem (2.0.0-alpha1): Extracting archive
  - Installing drupal/filehash (1.5.0): Extracting archive
  - Installing drupal/file_replace (1.2.0): Extracting archive
  - Installing drupal/features (3.12.0): Extracting archive
  - Installing drupal/eva (2.1.0): Extracting archive
  - Installing drupal/context (4.0.0-beta6): Extracting archive
  - Installing islandora/islandora (dev-8.x-1.x 85dae5c): Cloning 85dae5c9d8 from cache
  0/29 [>---------------------------]   0%
  8/29 [=======>--------------------]  27%
 17/29 [================>-----------]  58%
 27/29 [==========================>-]  93%
 29/29 [============================] 100%41 package suggestions were added by new dependencies, use `composer suggest` to see details.
Package doctrine/reflection is abandoned, you should avoid using it. Use roave/better-reflection instead.
Package silex/silex is abandoned, you should avoid using it. Use symfony/flex instead.
Generating autoload files
55 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
Scaffolding files for drupal/core:
  - Copy [project-root]/.editorconfig from assets/scaffold/files/editorconfig
  - Copy [project-root]/.gitattributes from assets/scaffold/files/gitattributes
./composer.json has been updated
Running composer update drush/drush
Loading composer repositories with package information
Updating dependencies
Lock file operations: 23 installs, 0 updates, 0 removals
  - Locking chi-teck/drupal-code-generator (1.33.1)
  - Locking consolidation/annotated-command (4.2.4)
  - Locking consolidation/config (1.2.1)
  - Locking consolidation/filter-via-dot-access-data (1.0.0)
  - Locking consolidation/log (2.0.2)
  - Locking consolidation/output-formatters (4.1.2)
  - Locking consolidation/robo (2.2.2)
  - Locking consolidation/self-update (1.2.0)
  - Locking consolidation/site-alias (3.1.0)
  - Locking consolidation/site-process (4.1.0)
  - Locking container-interop/container-interop (1.2.0)
  - Locking dflydev/dot-access-data (v1.1.0)
  - Locking drush/drush (10.3.6)
  - Locking grasmash/expander (1.0.0)
  - Locking grasmash/yaml-expander (1.4.0)
  - Locking league/container (2.5.0)
  - Locking nikic/php-parser (v4.11.0)
  - Locking psy/psysh (v0.10.8)
  - Locking symfony/filesystem (v4.4.26)
  - Locking symfony/finder (v5.3.0)
  - Locking webflo/drupal-finder (1.2.2)
  - Locking webmozart/assert (1.10.0)
  - Locking webmozart/path-util (2.3.0)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 23 installs, 0 updates, 0 removals
  - Downloading symfony/finder (v5.3.0)
  - Downloading dflydev/dot-access-data (v1.1.0)
  - Downloading consolidation/output-formatters (4.1.2)
  - Downloading consolidation/annotated-command (4.2.4)
  - Downloading consolidation/log (2.0.2)
  - Downloading symfony/filesystem (v4.4.26)
  - Downloading consolidation/self-update (1.2.0)
  - Downloading webmozart/assert (1.10.0)
  - Downloading webmozart/path-util (2.3.0)
  - Downloading webflo/drupal-finder (1.2.2)
  - Downloading nikic/php-parser (v4.11.0)
  - Downloading psy/psysh (v0.10.8)
  - Downloading container-interop/container-interop (1.2.0)
  - Downloading league/container (2.5.0)
  - Downloading grasmash/yaml-expander (1.4.0)
  - Downloading grasmash/expander (1.0.0)
  - Downloading consolidation/config (1.2.1)
  - Downloading consolidation/site-alias (3.1.0)
  - Downloading consolidation/site-process (4.1.0)
  - Downloading consolidation/robo (2.2.2)
  - Downloading consolidation/filter-via-dot-access-data (1.0.0)
  - Downloading chi-teck/drupal-code-generator (1.33.1)
  - Downloading drush/drush (10.3.6)
  0/23 [>---------------------------]   0%
 16/23 [===================>--------]  69%
 21/23 [=========================>--]  91%
 23/23 [============================] 100%  - Installing symfony/finder (v5.3.0): Extracting archive
  - Installing dflydev/dot-access-data (v1.1.0): Extracting archive
  - Installing consolidation/output-formatters (4.1.2): Extracting archive
  - Installing consolidation/annotated-command (4.2.4): Extracting archive
  - Installing consolidation/log (2.0.2): Extracting archive
  - Installing symfony/filesystem (v4.4.26): Extracting archive
  - Installing consolidation/self-update (1.2.0): Extracting archive
  - Installing webmozart/assert (1.10.0): Extracting archive
  - Installing webmozart/path-util (2.3.0): Extracting archive
  - Installing webflo/drupal-finder (1.2.2): Extracting archive
  - Installing nikic/php-parser (v4.11.0): Extracting archive
  - Installing psy/psysh (v0.10.8): Extracting archive
  - Installing container-interop/container-interop (1.2.0): Extracting archive
  - Installing league/container (2.5.0): Extracting archive
  - Installing grasmash/yaml-expander (1.4.0): Extracting archive
  - Installing grasmash/expander (1.0.0): Extracting archive
  - Installing consolidation/config (1.2.1): Extracting archive
  - Installing consolidation/site-alias (3.1.0): Extracting archive
  - Installing consolidation/site-process (4.1.0): Extracting archive
  - Installing consolidation/robo (2.2.2): Extracting archive
  - Installing consolidation/filter-via-dot-access-data (1.0.0): Extracting archive
  - Installing chi-teck/drupal-code-generator (1.33.1): Extracting archive
  - Installing drush/drush (10.3.6): Extracting archive
  0/13 [>---------------------------]   0%
  8/13 [=================>----------]  61%
 12/13 [=========================>--]  92%
 13/13 [============================] 100%7 package suggestions were added by new dependencies, use `composer suggest` to see details.
Package container-interop/container-interop is abandoned, you should avoid using it. Use psr/container instead.
Package doctrine/reflection is abandoned, you should avoid using it. Use roave/better-reflection instead.
Package silex/silex is abandoned, you should avoid using it. Use symfony/flex instead.
Generating autoload files
59 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
[cmd] with-contenv exited 0
[14-Jul-2021 14:07:53] NOTICE: Terminating ...
[14-Jul-2021 14:07:53] NOTICE: exiting, bye-bye!
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] waiting for services.
[services.d] service fpm finish: executing...
[services.d] service fpm exiting with exit code: 0
[services.d] service nginx finish: executing...
[services.d] service nginx exiting with exit code: 0
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
isle-dc_recast_1 is up-to-date
isle-dc_activemq_1 is up-to-date
isle-dc_homarus_1 is up-to-date
isle-dc_watchtower_1 is up-to-date
isle-dc_hypercube_1 is up-to-date
isle-dc_cantaloupe_1 is up-to-date
isle-dc_fits_1 is up-to-date
isle-dc_houdini_1 is up-to-date
isle-dc_blazegraph_1 is up-to-date
Recreating traefik ...
isle-dc_mariadb_1 is up-to-date
isle-dc_milliner_1 is up-to-date
isle-dc_alpaca_1 is up-to-date
isle-dc_solr_1 is up-to-date
isle-dc_fcrepo_1 is up-to-date
isle-dc_crayfits_1 is up-to-date
isle-dc_matomo_1 is up-to-date
Recreating traefik ... error

ERROR: for traefik  Cannot start service traefik: driver failed programming external connectivity on endpoint traefik (825e1f3b0b0b2d2977ae24835a5fd1b88feb87874a22278782524a6587c36957): Error starting userland proxy: listen tcp4 0.0.0.0:8082: bind: address already in use

ERROR: for traefik  Cannot start service traefik: driver failed programming external connectivity on endpoint traefik (825e1f3b0b0b2d2977ae24835a5fd1b88feb87874a22278782524a6587c36957): Error starting userland proxy: listen tcp4 0.0.0.0:8082: bind: address already in use
ERROR: Encountered errors while bringing up the project.
make: *** [Makefile:346: local] Error 1
arclight@grysbok:~/isle-dc>
```

**Expected results**
'make local' exits without an error and I'm able to start islandora with 'docker-compose up'.

**Comment by dannylamb** [comment]
Looks like port 8082 is already in use on your machine.  Is there something running on that port by default on centos? We can always bump the port number if we have to.

**Comment by mxMiles** [comment]
Thank you! There _was_ something running on that port. I killed several things (arclight, archivesspace, and httpd), ran `make local` again and it exited happily. Then, I ran `docker-compose up` and it looks like it's all working.

What's the proper way to bump the port number(s)? On an earlier attempt at installing, I edited `docker-compose.traefik.yml` so that the ports were like

```
    ports:
      - 10080:80       # drupal, cantaloupe, matomo
      - 10443:443     # https for ^^^
      ... etc
```
but that didn't seem to do the trick.
